Studies/NeuralNetEstimator/Common/JetNet.py

```diff
@@ -0,0 +1,63 @@
+import tensorflow as tf
+import pandas as pd
+import numpy as np
+import uproot
+
+from Common.JetNet_utils import MXLossFunc, H_WW_MXLossFunc
+
+
+class JetNet():
+    def __init__(self, features, labels, cfg):
+        self.features = features
+        self.labels = labels
+
+        # training parameters
+        self.lr = cfg['learning_rate']
+        self.n_epochs = cfg['n_epochs']
+        self.batch_size = cfg['batch_size']
+        self.verbosity = cfg['verbosity']
+        self.valid_split = cfg['valid_split']
+        self.name = cfg['name']
+        self.topology = cfg['topology']
+
+        self.model = None
+
+
+    def ConfigureModel(self, dataset_shape):
+        self.model = tf.keras.Sequential([tf.keras.layers.Dense(layer_size, activation='relu') for layer_size in self.topology])
```

Comment:
Define the model into a separate file and load it dynamically (based on what is specified in the config)